go/registry/api/runtime.go

```diff
@@ -262,3 +301,7 @@ type RuntimeGenesis struct {
 	// invalid iff the State is non-empty or StateRoot is an empty hash.
 	StorageReceipt signature.Signature `json:"storage_receipt"`
 }
+
+// TransactionSchedulerParameters is the transaction scheduler parameters.
+type TransactionSchedulerParameters struct {
```

Comment:
Since there is now a bunch of parameters I think it would be better to have multiple structs:
* compute parameters,
* merge parameters and
* txn scheduler parameters.

These would contain the group size/backup group size/etc. parameters instead of having everything directly inside the runtime descriptor.